prod/westeurope/internal/appbackend/autoscaling/terragrunt.hcl

```diff
@@ -84,13 +86,13 @@ inputs = {
           time_window        = "PT1M"
           time_aggregation   = "Average"
```

Comment:
```suggestion
          time_aggregation   = "Count"
```

Reply:
<img width="1525" alt="Screenshot 2020-12-02 at 12 18 46" src="https://user-images.githubusercontent.com/59291437/100866261-b0c05480-3498-11eb-8f6c-86cb1cdf4c8f.png">

Reply:
<img width="1525" alt="Screenshot 2020-12-02 at 12 18 52" src="https://user-images.githubusercontent.com/59291437/100866292-bae25300-3498-11eb-8d22-0a77ebff6282.png">

Reply:
with Average time_aggregation maybe spikes with 2K scale out too often

Reply:
OK .... let's try ..... and see how it goes  ;-)